p/scripts/extra.js

```diff
@@ -145,6 +145,9 @@ function init_archiving(parent) {
 const freshrssSliderLoadEvent = new Event('freshrss:slider-load');
 
 function open_slider_listener(ev) {
+	if (ev.ctrlKey || ev.shiftKey) {
+		return;
+	}
```

Comment:
@Frenzie Why is is important to respect the `Shift` key?

Reply:
That's new window.